nexus_constructor/instrument_view/qentity_utils.py

```diff
@@ -49,3 +61,116 @@ def create_qentity(
     for component in components:
         entity.addComponent(component)
     return entity
+
+
+class NeutronSource:
```

Comment:
maybe having this in the geometry package would be better (see slit and disk_chopper).